teachertool/src/components/RubricWorkspace.tsx

```diff
@@ -53,7 +54,11 @@ const WorkspaceTabButtons: React.FC = () => {
     );
 };
 
-const WorkspaceTabPanels: React.FC = () => {
+interface PrintRefProps {
```

Comment:
Lets stick with bespoke Props interfaces, for consistency.
```suggestion
interface WorkspaceTabPanelsProps {
```